02-esc/modal.js

```diff
@@ -58,54 +64,52 @@ function openModal() {
         `;
     
     function getBookingInformationFromForm() {
-
         customerName = document.querySelector('#name').value;
         eMail = document.querySelector('#e-mail').value;
         time = document.querySelector('#time').value;
         participants = parseInt(document.querySelector('#participants').value);
     }
-    
          
     let openModalStepThreeBtn = document.querySelector(".open-modal-step-3");
     openModalStepThreeBtn.addEventListener("click", getBookingInformationFromForm);
     openModalStepThreeBtn.addEventListener("click", makeBooking);
-    
     openModalStepThreeBtn.addEventListener("click", openModalStepThree);
 
     function openModalStepThree(e) {
-
         e.preventDefault();
-
         modalDiv.innerHTML = `
         <div class="bookingStep3Content">
             <h1>Thank you!</h1>
             <a href="">Back to challenges</a>
          </div>
         `;
-
     }
 }
 }
 
-
 async function makeBooking() {
-
 const res = await fetch('https://lernia-sjj-assignments.vercel.app/api/booking/reservations', {
     method: 'POST',
     headers: {
         'Content-Type': 'application/json',
     },
     body: JSON.stringify({
-        challenge: 12,
+        challenge: idVariabelForModal,
         name: customerName,
         email: eMail,
-        date: "2022-12-12",
+        date: date,
         time: time,
         participants: participants,
     }),
 });
 const data = await res.json();
 console.log(data);
-
 }
 
+async function checkAvailableTimes(selectedDate, challengeId) {
+    const res = await fetch(`https://lernia-sjj-assignments.vercel.app/api/booking/available-times?date=${selectedDate}&challenge=${challengeId}`);
+    const data = await res.json();
+    data.slots.forEach(slot => {
+        console.log(slot) 
+})
+}
```

Comment:
Bra början! Fundera på vad ni ska göra med informationen ni får tillbaka och hur ni ska få in den i nästa steg 😉

Reply:
Yes!